File: peek_plugin_unified_search/_private/tuples/StringCapToggleActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_unified_search._private.PluginNames import unifiedSearchTuplePrefix


@addTupleType
class StringCapToggleActionTuple(TupleActionABC):
    __tupleType__ = unifiedSearchTuplePrefix + "StringCapToggleActionTuple"

    excludedKwId = TupleField()

File: peek_plugin_unified_search/plugin-module/_private/tuples/StringCapToggleActionTuple.ts
                                       import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {unifiedSearchTuplePrefix} from "../PluginNames";

@addTupleType
export class StringCapToggleActionTuple extends TupleActionABC {
    static readonly tupleName = unifiedSearchTuplePrefix + "StringCapToggleActionTuple";

    excludedKwId: number;

    constructor() {
        super(StringCapToggleActionTuple.tupleName)
    }
}